calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -74,14 +74,18 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
 ### Enable WAF
 
 1. On the Manager UI, click **Threat Defense**, **Web Application Firewall**, **Configure Web Application Firewall**.
-2. For each service that you want to enable for WAF, click the **Action** column, select Enable.
-
-  <img
-    src='/img/calico-enterprise/waf-services.png'
+2. On the Select Services to Inspect page, choose all the services for which you want WAF to inspect the traffic.
+    
+   <img
+    src='/img/calico-enterprise/waf-select-services.png'
     alt='WAF services'
     width='600'
   />
 
+:::note
```

Comment:
I'd add this as step 3. and add an image of the 3rd page maybe ?

Reply:
@LorcanMcVeigh it's done